externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/WithSparkSQLEngine.scala

```diff
@@ -35,7 +35,9 @@ trait WithSparkSQLEngine extends KyuubiFunSuite {
   // Behavior is affected by the initialization SQL: 'SHOW DATABASES'
   // SPARK-35378 (3.2.0) makes it triggers job
   // SPARK-43124 (4.0.0) makes it avoid triggering job
-  protected val initJobId: Int = if (SPARK_ENGINE_RUNTIME_VERSION >= "4.0") 0 else 1
+  // KYUUBI #6789 makes it avoid triggering job
+  // protected val initJobId: Int = if (SPARK_ENGINE_RUNTIME_VERSION >= "4.0") 0 else 1
+  protected val initJobId: Int = 0
```

Comment:
how do you think about? @pan3793

Reply:
Fixed by @wForget  https://github.com/apache/spark/pull/45397

Reply:
In our use case, I set initial/mini executors to 0 for notebook connections.

If job triggered by initialization SQL, it will need at least 1 executor before the connection ready.

Reply:
And for notebook connections, we will init the spark driver in a temporary queue, and then move it to the final user queues, so it should be fast to make the connection ready even the user queue has no available resources.

But if job triggered during initialization, it will make it slow if the  user queue has no available resources.